recipe/meta.yaml

```diff
@@ -40,41 +43,41 @@ requirements:
     - python
     - pip
     - cython
-    - arrow-cpp
+    - arrow-cpp >=8,<9
     - pyarrow
     - numpy
     - setuptools >=40.6.0
     - wheel
   run:
     - python
-    - {{ pin_compatible('pyarrow', max_pin='x') }}
```

Comment:
I rearranged the `run` dependencies to more closely match those found in setup.cfg:
https://github.com/snowflakedb/snowflake-connector-python/blob/main/setup.cfg#L44-L59

Note the absence of `pyarrow` or `arrow-cpp`. This is because they make that a build dependency only, so they can vendor the relevant libraries above.

Reply:
OK